test/routing/DeparturesRouterSpec.scala

```diff
@@ -267,4 +267,50 @@ class DeparturesRouterSpec extends AnyFreeSpec with Matchers with OptionValues w
       }
     }
   }
+
+  "when submitting a movements departure" - {
```

Comment:
That's not quite what we're doing here - we're submitting a new message for an existing departure.

Reply:
done